setup_eunit.template

```diff
@@ -1,6 +1,6 @@
 {variables, [
     {package_author_name, "The Apache Software Foundation"},
-    {cluster_port, 5984},
+    {cluster_port, 0},
```

Comment:
Why you didn't change backend port as well? If you have port assign conflict, you solved it for chttpd, but your httpd service will crash causing further issues with his interaction.